documentation/assemblies/deploying/assembly-kraft-mode.adoc

```diff
@@ -50,11 +50,42 @@ In the following example, three nodes perform a dual role and two nodes act only
 .Example cluster with dual-role nodes and dedicated broker nodes
 image::kraft-dual-role-quorum.png[KRaft cluster with nodes that combine roles]
 
-== KRaft limitations
+== Controller scaling
 
-Currently, the KRaft mode in Strimzi has the following major limitations:
+KRaft mode supports two types of controller quorums:
 
-* Scaling of KRaft controller nodes up or down is not supported.
+* *Static controller quorums* +
+In this mode, the number of controllers is fixed, and scaling requires downtime.
+* *Dynamic controller quorums* +
+This mode enables dynamic scaling of controllers without downtime. 
+New controllers join as observers, replicate the metadata log, and eventually become eligible to join the quorum.
+If a controller being removed is the active controller, it will step down from the quorum only after the new quorum is confirmed. 
+
+Scaling is useful not only for adding or removing controllers, but supports the following operations:
+
+* Replacing controllers because of hardware failure
+* Migrating clusters onto new hardware
+* Moving nodes from dedicated controller roles to dual roles or vice versa
+
+Dynamic controller quorums provide the flexibility to make these operations significantly easier to perform.
+
+== Limitations with static controller quorums
+
+Strimzi currently supports only static controller quorums.
+This limitation means dynamic scaling of controller quorums cannot be used to support the following:
+
+* Scaling up or down of node pools that have controller roles
+* Adding or removing controller roles from node pools
+* Switching node pools between dedicated controller roles and dual roles
```

Comment:
This is not about what Strimzi supports. It is about what Kafka does not support and should be formulated in that way. I would use something like this.

```suggestion
Migration between static and dynamic controller quorums is not currently supported by Apache Kafka, though it is expected to be introduced in a future release.
All pre-existing KRaft-based Apache Kafka clusters that use the static quorum have to keep using it.
In order to keep supporting existing KRaft-based clusters, Strimzi has to keep using the static quorum as well.

This limitation means dynamic scaling of controller quorums cannot be used to support the following:
* Adding or removing node pools with controller roles
* Adding the controller role to an existing node pool
* Removing the controller role from an existing node pool
* Scaling a node pool with the controller role
* Renaming a node pool with the controller role
```